.gitignore

```diff
@@ -137,10 +138,13 @@ pytest.ini
 
 # Sensible data
 config.json
+local.json
 secrets
 
 # Custom
 /data
+/otri/data
```

Comment:
Isn't this included in every "/data" folder name? Why would you duplicate it?

Reply:
As a matter of fact I'm not even sure why is this here, don't remember adding it